content/en/admin/moderation.md

```diff
@@ -45,6 +45,8 @@ Once the data has been deleted, whether that is after the 30-day period, or if a
 
 For remote accounts, suspending will make them unfollow any local account. Those relationships are not restored in case the remote account is unsuspended, even within the 30-day time window.
 
+Note that by default, users suspended by an instance will still be able to view posts from that instance. To change this default behavior, instance admins can set the `AUTHORIZED_FETCH` environment variable - see the documentation for [configuring your environment](../config/).
```

Comment:
please reword instance -> server (to match the rest of the page, and the [style guide](https://github.com/mastodon/documentation/blob/main/DOCSTYLE.md)